.env

```diff
@@ -1 +1,3 @@
-SINGULARITY_TAG=v0.3.1
\ No newline at end of file
+SINGULARITY_TAG=v0.3.1
+LOTUS_API=http://lotus:1234/rpc/v1
+LOTUS_TOKEN=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJBbGxvdyI6WyJyZWFkIiwid3JpdGUiLCJzaWduIiwiYWRtaW4iXX0.HaorU9TFqN7dIOeM88U9gomBpE1JlcZbF6Ybm0w17A8
```

Comment:
Can you remove this please?

I use `direnv` with a local `.envrc` that's ignored by git for this sort of thing which I find it works OK

Reply:
don't worry, it's a useless throw away token anyway :)